README.md

```diff
@@ -126,9 +136,12 @@ e.g. `docker run -v /config:/config  ghcr.io/caas-team/sparrow --config /config/
 
 ## Configuration
 
-The configuration is divided into two parts. The startup configuration and the runtime configuration. The startup
-configuration is a technical configuration to configure the `sparrow` instance itself. The runtime configuration will be
-loaded by the `loader` from a remote endpoint. This configuration consists of the checks' configuration.
+The configuration is divided into two parts. The startup configuration and the checks configuration. The startup
+configuration is a technical configuration to configure the `sparrow` instance itself.
+
+The checks configuration will be
+loaded during runtime by the `loader` from a remote endpoint. The `loader` can be used to get the configuration local as well (type: `file`).
```

Comment:
"using a `file` loader, you may load the checks configuration for a local file directly." - let's try to use use direct wording istread of passive voice